Antidote/AbstractCallViewController.h

```diff
@@ -0,0 +1,34 @@
+//
+//  AbstractCallViewController.h
+//  Antidote
+//
+//  Created by Chuong Vu on 7/2/15.
+//  Copyright (c) 2015 dvor. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@class OCTSubmanagerCalls;
+@class OCTChat;
+@class OCTCall;
+
+@interface AbstractCallViewController : UIViewController
+
+/**
+ * Create an instance of the CallViewController.
+ * @param chat Appropriate chat for the call.
+ * @param manager The call maanger for the application.
+ */
+- (instancetype)initWithChat:(OCTChat *)chat submanagerCalls:(OCTSubmanagerCalls *)manager;
+
+/**
+ * The call associated with this view.
```

Comment:
Also you can mention that it is autoupdating call. It would be nice to have public method `- (void)didUpdateCall NS_REQUIRES_SUPER;`
- which will be called by abstract controller after updating call
- abstract controller can do stuff some stuff in this method if it like
- subclass can override it if it like and do some other stuff.